api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -48,8 +48,10 @@ message ExtAuthz {
 //
 // A failed check will cause this filter to close the HTTP request normally with 403 (Forbidden),
 // unless a different status code has been indicated by the authorization server via response
-// headers. If other headers in the authorization response need to be sent to client, this can also
-// be done by specifying them in *allowed_authorization_headers*.
+// headers. Note that if any of the 5xx HTTP error codes or an error is originated during this
+// checking, the filter will also return a Forbidden response to the client and failures will be
+// visible in the logs. If other headers in the authorization response need to be sent to client,
```

Comment:
I don't think using logs is a reasonable way to expect users to be able to monitor behavior. They should have a stats based option for monitoring systems; at scale you don't want to have to be emitting lots of log data and transforming this back to structured data.